### PR TITLE
Brev karakterutskrift journalføring

### PR DESCRIPTION
**Pull request by ViktorGSolberg**

### CODE REVIEW

File: src/main/kotlin/no/nav/familie/ef/sak/iverksett/IverksettClient.kt [hunk 1]
```diff
@@ -105,4 +106,8 @@ class IverksettClient(
     fun sendFrittståendeBrev(frittståendeBrevDto: FrittståendeBrevDto) {
         postForEntity<Any>(URI.create("$familieEfIverksettUri/api/brev/frittstaende"), frittståendeBrevDto)
     }
+
+    fun håndterUtsendingAvKarakterutskriftBrev(karakterutskriftBrevDto: KarakterutskriftBrevDto) {
+        postForEntity<Any>(URI.create("$familieEfIverksettUri/api/brev/frittstaende/innhenting-karakterutskrift"), karakterutskriftBrevDto)
```
Comment (eirarset) [inline src/main/kotlin/no/nav/familie/ef/sak/iverksett/IverksettClient.kt:111]: Kanskje 🤔 
```suggestion
        postForEntity<Unit>(URI.create("$familieEfIverksettUri/api/brev/frittstaende/innhenting-karakterutskrift"), karakterutskriftBrevDto)
```

File: src/main/kotlin/no/nav/familie/ef/sak/karakterutskrift/SendKarakterutskriftBrevTilIverksettTask.kt [hunk 2]
```diff
@@ -25,18 +30,20 @@ import java.util.Properties
 
 @Service
 @TaskStepBeskrivelse(
-    taskStepType = KarakterutskriftBrevTask.TYPE,
+    taskStepType = SendKarakterutskriftBrevTilIverksettTask.TYPE,
```
Comment (eirarset) [inline src/main/kotlin/no/nav/familie/ef/sak/karakterutskrift/SendKarakterutskriftBrevTilIverksettTask.kt:33]: Burde nok også oppdatere TYPE

Comment (ViktorGSolberg) [reply src/main/kotlin/no/nav/familie/ef/sak/karakterutskrift/SendKarakterutskriftBrevTilIverksettTask.kt:33]: fixd [12e81f9](https://github.com/navikt/familie-ef-sak/pull/2246/commits/12e81f93706e8a9a3cdfcb0d829d0972f93536df)

File: src/main/kotlin/no/nav/familie/ef/sak/iverksett/IverksettClient.kt (hunk 1 above)
Comment (ViktorGSolberg) [reply src/main/kotlin/no/nav/familie/ef/sak/iverksett/IverksettClient.kt]: fixd [12e81f9](https://github.com/navikt/familie-ef-sak/pull/2246/commits/12e81f93706e8a9a3cdfcb0d829d0972f93536df)

**Review by eirarset — APPROVED** [review]

**Pull request merged**